Userland/Libraries/LibCore/EventReceiver.cpp

```diff
@@ -129,10 +129,7 @@ void EventReceiver::stop_timer()
 {
     if (!m_timer_id)
         return;
-    bool success = Core::EventLoop::unregister_timer(m_timer_id);
-    if (!success) {
-        dbgln("{:p} could not unregister timer {}", this, m_timer_id);
-    }
```

Comment:
This log message actually used to occur in AudioServer and/or AudioClient (?) on Serenity, because it unregistered timers on one thread that were registered in another. Have you verified that we don't get that bad behavior anymore? or are we declaring it not a problem anymore with the refactor.

Reply:
Can't seem to trigger this message on master. I tried playing an mp3 in Sound Player and in Ladybird and then logging out and/or powering off.
Either way, unregistering timers in a wrong thread is a very bad idea even with this safeguard.